sisl/io/tests/test_object.py

```diff
@@ -56,6 +64,24 @@ class TestObject(object):
     def test_siesta_sources(self):
         pytest.importorskip("sisl.io.siesta._siesta")
 
+    def test_direct_path_instantiation(self):
+        fp = Path("/tmp/shouldnotexist.1234567")
```

Comment:
use `Path("tmp") / Path("...")`, since we may support windows in the not too far future. `osp.join` or `Path` intrinsic should be used